scripts/init.js

```diff
@@ -23,9 +23,14 @@ module.exports = function(appPath, appName, verbose, originalDirectory) {
     appPackage.dependencies[key] = ownPackage.devDependencies[key];
   });
 
+  // Test config
+  appPackage.jest = {
+    preset: '<rootDir>/node_modules/react-scripts/config/jest',
```

Comment:
We had to do it for ESLint too for the sake of editor integration. I guess it's unavoidable on some level, but I'd like these to stay "pointers" and no more than a couple of lines per tool max.